assets/translations/en.json

```diff
@@ -63,5 +63,7 @@
 
     "crash_manager" : {
         "activated_label" : "Crash manager is activated:"
-    }
+    },
+
+    "label_force_language_setting" : "langage is english"
```

Comment:
This translation is not used right ?

Reply:
it's the translation for the firebase remote config key